src/store/market/market-state.ts

```diff
@@ -0,0 +1,22 @@
+import { MarketCoinsSortFieldEnum } from '../../enums/market-coins-sort-field.enum';
+import { LoadableEntityState } from '../types';
+import { createEntity } from './../create-entity';
+import { MarketCoin } from './market.interfaces';
+
+export interface MarketState {
+  favouriteTokens: Array<string>;
+  sortField: MarketCoinsSortFieldEnum;
+  tokens: LoadableEntityState<Array<MarketCoin>>;
```

Comment:
something with naming (you call 1 thing with 2 different names)  
**tokens**: LoadableEntityState<Array<Market**Coin**>>;